lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -84,8 +84,9 @@ def perform(form526_submission_id)
         job_status.update(status: Form526JobStatus::STATUS[:success])
         submission.deliver_to_backup!
       rescue => e
-        ::Rails.logger.error(
-          message: "FORM526 BACKUP SUBMISSION FAILURE. Investigate immediately: #{e.message}.",
+        ::Rails.logger.warn(
+          message: 'Form 526 backup submission failure. retrying...',
```

Comment:
Only suggestion would be to reference the class explicitly to potentially aid in error investigation efforts.